gradle-versions-plugin/src/main/kotlin/com/github/benmanes/gradle/versions/updates/DependencyUpdates.kt

```diff
@@ -21,18 +21,18 @@ import org.gradle.api.specs.Spec
 class DependencyUpdates
   @JvmOverloads
   constructor(
-    val project: Project,
-    val resolutionStrategy: Action<in ResolutionStrategyWithCurrent>?,
-    val revision: String,
+    private val project: Project,
```

Comment:
I’m doubtful, but an extension plugin could have used this constructor if public. It might not be worth worrying about though

Reply:
https://github.com/fkorotkov/gradle-libraries-plugin/blob/92311acf0cdd7019f7ba7fb55d7c70e98acde9ec/src/main/java/com/fkorotkov/gradle/libraries/tasks/LibrariesBaseTask.java#L47

Reply:
Reverted.